doc/whats_new/v0.21.rst

```diff
@@ -20,6 +20,13 @@ random sampling procedures.
 - :class:`linear_model.BayesianRidge` |Fix|
 - Decision trees and derived ensembles when both `max_depth` and
   `max_leaf_nodes` are set. |Fix|
+- :func:`decomposition.sparse_encode()` with `algorithm='lasso_lars'` |Fix|
```

Comment:
Technically, we only say this listing relates to estimators ...?

Reply:
> Technically, we only say this listing relates to estimators ...?

ok. removed them.